pipeline/config/exoscale/sc-config.yaml

```
@@ -415,6 +415,8 @@ fluentd:
     affinity: {}
     nodeSelector: {}
     useRegionEndpoint: true
+    chunkLimitSize: 2048
+    queueLimitSize: 4096
```

I think this makes it a bit more clear on what they "represent"
```suggestion
    chunkLimitSizeBytes: 2048
    queueLimitSizeBytes: 4096
```